osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/config/ApplicationConfig.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.secretmanagement.application.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource("classpath:osgp-secret-management.properties")
+@PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true)
+@PropertySource(value = "file:${osgp/SecretManagement/config}", ignoreResourceNotFound = true)
+public class ApplicationConfig {
+    @Value("${component.name}")
+    public static final String COMPONENT_NAME = "OSGP-SECRET-MANAGEMENT";
```

Comment:
Use @Value("${component.name:OSGP-SECRET-MANAGEMENT}") to provide default value.
Also, final fields should be set through constructor only. Best to remove final and I would probably make this field private.